src/main/resources/db/migration/V11.02__add_initial_accounts_with_encrypted_pass.sql

```diff
@@ -0,0 +1,7 @@
+SET @current_date = NOW();
```

Comment:
No need of new migration file, just put all code from the task in one migration file